languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,7 +10069,43 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
-    <rule id="LA_OU_LES_PLUS" name="la ou les plus" default="temp_off">
+      <rulegroup id="ELLES_VERBE_TOUTES" name="elles verbe toutes" default="temp_off">
+          <rule>
+              <pattern>
+                  <token postag="R.* f p" postag_regexp="yes"/>
+                  <token postag="V.*" postag_regexp="yes"/>
+                  <marker>
+                      <token postag="A" regexp="yes">tout(e)?</token>
+                  </marker>
+                  <token postag="D.*" postag_regexp="yes"><exception regexp="yes">un|la</exception></token>
+                  <token postag="N.*" postag_regexp="yes"><exception regexp="yes">temps|droit?|an?|jour?|année?</exception></token>
```

Comment:
This seems strange: `temps|droit?|an?|jour?|année?`. It will match: temps, droit, droi, an, a, jour, jou, année, anné